WordPress/src/main/res/layout/quick_start_completed_tasks_list_header.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical">
+
+    <View
+        android:id="@+id/top_spacing"
+        android:layout_width="wrap_content"
+        android:layout_height="@dimen/margin_extra_large"
+        android:background="@color/grey_lighten_30"/>
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="48dp"
```

Comment:
```suggestion
        android:layout_height="wrap_content"
        android:minHeight="@dimen/one_line_list_item_height"
```

Can we replace the hard-coded `48dp` value with a dimension reference like `one_line_list_item_height`?  Also, using `android:layout_height="wrap_content"` and `android:minHeight="@dimen/one_line_list_item_height"` will  give us the `48dp` layout height for most cases and expand the view height to avoid clipping the text in case the user has a large text settings enabled.